src/main/java/mate/academy/dao/impl/UserDaoImpl.java

```diff
@@ -26,7 +26,8 @@ public User add(User user) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new RuntimeException("Can't save user in DB", e);
+            throw new RuntimeException("Can't save user with email="
+                    + user.getEmail() + " in DB", e);
```

Comment:
Why do we have to receive excessive info about user in this message? I believe just user's login (an email in this case) would be enough. I'll correct this but I'm not sure if it's really necessary

Reply:
sometimes it is essential to know all info about an object which can't be saved to DB. Because the reason can be in the content of some field. But you don't know what kind of field it might be.